dev-aws/kafka-shared-msk/contact-channels/account-identity.tf

```diff
@@ -0,0 +1,8 @@
+# Consume from account-identity.legacy.account.events.compacted
+module "legacy_account_events_consumer" {
+  source           = "../../../modules/tls-app"
+  cert_common_name = "contact-channels/legacy-account-events-consumer"
+  consume_topics   = ["account-identity.legacy.account.events.compacted"]
+  consume_groups   = ["contact-channels.legacy-account-events-consumer"]
```

Comment:
Is this a new consumer group or a migrated one?

Reply:
@sbuliarca this is a consumer we are migrating from proximo to this, but technically it will be a new one because we are moving to a new topic and reading from the beginning.